core/controllers/PhotoController.js

```diff
@@ -0,0 +1,15 @@
+import Photo from '../models/Photo.js';
+
+class PhotoController {
+  async getPhotosByAuthor(req, res, next) {}
+  async getPhotosPublicByAuthor(req, res, next) {}
+  async getLikesById(req, res, next) {}
```

Comment:
The PhotoSchema already has `like` attr, should we have this route?

Reply:
router.get('/:user-id/', photoController.getPhotosByAuthor); Dùng để lấy danh sách photo (private và public) của 1 user bất kỳ theo id.
router.get('/public/:user-id/', photoController.getPhotosPublicByAuthor); Dùng để lấy danh sách photo (chỉ public) của 1 user bất kỳ theo id.
router.get('/like/:photo-id/', photoController.getLikesById); Dùng để hiển thị số like của 1 photo(icon trái tym).